sci-geosciences/grass/grass-7.8.7-r2.ebuild

```diff
@@ -116,7 +116,10 @@ src_prepare() {
 	sed -e "s:= python3:= ${EPYTHON}:" -i "${S}/include/Make/Platform.make.in" || die
 
 	default
-	eautoreconf
+
+	# Avoid call eautoreconf or eautoheader here. The file config.in.h is
+	# maintained manually upstream. Changes to it may lead to undefined
+	# behavior. See bug #866554.
```

Comment:
Aha! Okay, so we should put a commented out example with AT_NOEAUTOHEADER=1 eautoreconf :)

thanks for explaining!

Reply:
By the way: that variable was added because of a disaster with libsdl2 similar to this!